socrata_filter/tests/src/Kernel/FilterSocrataTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Drupal\Tests\socrata_filter\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\socrata\Entity\Endpoint;
+use Drupal\filter\FilterPluginCollection;
+use Drupal\Core\Render\RenderContext;
+
+/**
+ * Endpoint units tests.
+ *
+ * @group socrata_filter
+ */
+class FilterSocrataTest extends KernelTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = ['socrata_filter', 'socrata', 'filter'];
+
+  /**
+   * Filter interface.
+   *
+   * @var \Drupal\filter\Plugin\FilterInterface[]
+   */
+  protected $filter;
+
+  /**
+   * Endpoint object.
+   *
+   * @var object
+   */
+  protected $endpoint;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+
+    $data = [
+      'url' => 'https://data.seattle.gov/resource/rn6u-vkuv.json',
+      'id' => 'endpoint',
+      'label' => 'Endpoint',
+      'app_token' => '',
+    ];
```

Comment:
I moved config to a YAML file in a /tests/config dir in the main module. I don't think the /config/install dir would be appropriate, nor using the config API, I think that's really for module settings configuration.